src/peep/peep.c

```diff
@@ -8399,12 +8444,12 @@ static bool peep_should_use_cash_machine(rct_peep *peep, int rideIndex)
  *
  *  rct2: 0x0069A98C
  */
-static void sub_69A98C(rct_peep *peep)
+static void reset_pathfind_goal(rct_peep *peep)
```

Comment:
This function is used by both guests and staff so should really be prepended `peep_`.